docs/breaking-changes.asciidoc

```diff
@@ -0,0 +1,12 @@
+[[breaking-changes]]
+== Breaking changes
```

Comment:
The "c" should be capitalized in changes so it matches the other section titles in the table of contents